tools/cli/wskaction.py

```diff
@@ -170,7 +174,12 @@ def getExec(self, args, props):
             exe = self.getActionExec(args, props, pipeAction)
         elif args.artifact is not None and os.path.isfile(args.artifact):
             contents = open(args.artifact, 'rb').read()
-            if args.artifact.endswith('.swift'):
+            # FIXME:
```

Comment:
type should not assume `swift3` but rather accept an additional value to be used as the type.